### DISCUSSION TITLE
Who is using Simple-OpenAI?

### DISCUSSION THREAD

**Discussion by sashirestela** [sashirestela/simple-openai#171] [General]
- [ChatMotor](https://www.chatmotor.ai/): A framework for OpenAI services. Thanks for [credits](https://docs.chatmotor.ai/rest/soft/1.0-BETA/user-guide/User-Guide.html#credits-and-acknowledgments)!
- [OpenOLAT](https://github.com/OpenOLAT/OpenOLAT): A learning managment system.
- [Willy](https://github.com/laravieira/Willy): A multi-channel assistant.
- [SuperTurtyBot](https://github.com/DaRealTurtyWurty/SuperTurtyBot): A multi-purpose discord bot.
- [Woolly](https://github.com/da-z/woolly): A code generation IntelliJ plugin.
- [Vinopener](https://github.com/thevinopener/vinopener): A wine recommender app.
- [ScalerX.ai](https://scalerX.ai): A Telegram chatbot factory.

**Comment by amavashev** [comment]
Please update Cryptk.ai GPT Chatbot to scalerX.ai - Launch, Train RAG Agents (project was renamed), here is official website https://scalerX.ai.
